Description: Fix compilation with gcc 10
 Tweak source to compile with gcc 10.
Bug-Debian: https://bugs.debian.org/957418
Author: dod
Applied-Upstream: https://github.com/lcdproc/lcdproc/commit/084126b03f29b33ab3e657c66e8c6a439f27f8e1
--- a/clients/lcdproc/iface.h
+++ b/clients/lcdproc/iface.h
@@ -18,7 +18,7 @@
 /** max number of interfaces in multi-interface mode */
 #define MAX_INTERFACES 3
 
-IfaceInfo iface[MAX_INTERFACES];	/* interface info */
+IfaceInfo iface[MAX_INTERFACES] __attribute__((common));	/* interface info */
 
 /** Update screen content */
 int iface_screen(int rep, int display, int *flags_ptr);
